app/app.go

```diff
@@ -369,60 +360,51 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	app.GroupKeeper = groupkeeper.NewKeeper(keys[group.StoreKey], appCodec, app.MsgServiceRouter(), app.AccountKeeper, groupConfig)
 
 	// register custom modules here
-	app.smm = setCustomModules(app, interfaceRegistry)
-	ecocreditModule := ecocreditmodule.NewModule(
+	ecocreditMod := ecocreditmodule.NewModule(
+		app.keys[ecocredit.ModuleName],
 		app.GetSubspace(ecocredit.DefaultParamspace),
 		app.AccountKeeper,
 		app.BankKeeper,
 		authtypes.NewModuleAddress(govtypes.ModuleName),
 	)
-	dataModule := data.NewModule(app.AccountKeeper, app.BankKeeper)
-	newModules := []moduletypes.Module{ecocreditModule, dataModule}
-	err := app.smm.RegisterModules(newModules)
-	if err != nil {
-		panic(err)
-	}
-	err = app.smm.CompleteInitialization()
-	if err != nil {
-		panic(err)
-	}
-	app.smm.RegisterInvariants(&app.CrisisKeeper)
+	dataMod := datamodule.NewModule(app.keys[data.ModuleName], app.AccountKeeper, app.BankKeeper)
 
 	govConfig := govtypes.DefaultConfig()
 	app.GovKeeper = govkeeper.NewKeeper(
 		appCodec, keys[govtypes.StoreKey], app.GetSubspace(govtypes.ModuleName), app.AccountKeeper, app.BankKeeper,
 		&stakingKeeper, govRouter, app.MsgServiceRouter(), govConfig,
 	)
 
-	app.setCustomKeepers(bApp, keys, appCodec, govRouter, homePath, appOpts)
+	// TODO(Tyler): not needed??
+	//app.setCustomKeepers(bApp, keys, appCodec, govRouter, homePath, appOpts)
```

Comment:
I think there are a few of these that we included but don't use. This as well as `setCustomKVStoreKeys` in view.

Reply:
I'm in favor of removing what we aren't using and don't see a need for anytime soon.

Reply:
all empty calls are now removed 👍🏻